internal/pubsub/pubsub.go

```diff
@@ -193,23 +188,28 @@ func (s *Subscription) Close() error {
 	return s.driver.Close()
 }
 
+// ackIDBox makes it possible to use a driver.AckID with bundler.
+type ackIDBox struct {
```

Comment:
Ah, I think I see why you need this. Unfortunate.

Reply:
Yep.